internal/globalratelimit/runner/runner.go

```diff
@@ -120,6 +124,7 @@ func (r *Runner) subscribeAndTranslate(ctx context.Context) {
 					r.Logger.Error(err, "failed to update the config snapshot")
```

Comment:
is `r.addNewSnapshot(ctx, nil)` still valid ? looks like it will need to be replaced by `r.updateSnapshot(ctx, xdsResourcesToUpdate)` ?

Reply:
🤦‍♂️